plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/FileBasedWorkspaceInitializer.java

```diff
@@ -57,6 +58,9 @@
 	@Inject
 	private FileBasedWorkspace workspace;
 
+	@Inject
+	private N4JSConfigSnapshotFactory configSnapshotFactory;
```

Comment:
Probably this won't cause problems, but I think it would be better to use `ConfigSnapshotFactory` as type, here.

Reply:
Same is done with `N4JSScopeProvider` at several locations, e.g. in `N4JSPostProcessor`.